modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -371,12 +387,13 @@ internal class DiscoverAdapter(
                             btnSubscribe.setOnClickListener {
                                 btnSubscribe.updateSubscribeButtonIcon(true)
                                 listener.onPodcastSubscribe(podcast = podcast, listUuid = row.listUuid)
+                                row.listUuid?.let { listUuid -> trackDiscoverListPodcastSubscribed(listUuid, podcast.uuid) }
```

Comment:
Nice catch here. This fixes a bug where we weren't sending this event if the `+` was clicked on a sponsored podcast.